Sources/ArcGISToolkit/Documentation.docc/Resources/FlyoverSceneView/FlyoverSceneViewStep2.swift

```diff
@@ -0,0 +1,26 @@
+import SwiftUI
+import ArcGIS
+import ArcGISToolkit
+
+struct FlyoverExampleView: View {
+    @State private var scene = Scene(
+        item: PortalItem(
+            portal: .arcGISOnline(connection: .anonymous),
+            id: PortalItem.ID("7558ee942b2547019f66885c44d4f0b1")!
+        )
+    )
+    
+    var body: some View {
+        FlyoverSceneView(
+            initialLocation: Point(
+                x: 4.4777,
+                y: 51.9244,
+                z: 1_000,
+                spatialReference: .wgs84
+            ),
+            translationFactor: 1_000
+        ) { sceneViewProxy in
```

Comment:
```suggestion
        ) { _ in
```
Should the proxy be named here? Naming it seems helpful to indicate that the proxy is available.

Reply:
I don't think so if it is not being used. They can see the proxy is available by reading the view init or they can see it via code completion.